build.sbt

```diff
@@ -12,6 +12,7 @@ val common = library("common")
       awsCore,
       awsCloudwatch,
       awsDynamodb,
+      awsKinesis,
```

Comment:
NB: I think we'll also need to explicitly define the dependency for ec2 now as well. It's required here https://github.com/guardian/frontend/blob/main/admin/app/controllers/admin/TroubleshooterController.scala#L3, so we need to add `val awsEc2 = "com.amazonaws" % "aws-java-sdk-ec2" % awsVersion` to Dependencies.scala